crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -134,7 +131,10 @@ def transition(inst, name, state)
           # it in lockstep.  Naturally the second save clobbers
           # the first, so the first node won't be present in that
           # proposal.
-          bc_lock = acquire_lock "#{bc}:#{rname}"
+          file_lock = Crowbar::Lock.new(
+            logger: @logger,
+            path: Rails.root.join("tmp", "#{bc}:#{rname}.lock")
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.